actionlist/__tests__/components/sideMenuSpec.js

```diff
@@ -13,19 +13,22 @@ jest.unmock('../../app/components/SideMenu');
 describe('SideMenu', () => {
   it('should render the inner tree component structure properly', () => {
     const middlewares = [];
-    const initialState = {};
+    const initialState = {
+      actionItemsReducer: {
```

Comment:
The initial state is the fake state we are passing to the component and verifying whether the data is correctly being retrieved by inner react native components or not.